src/handlers/rfqm_handlers.ts

```diff
@@ -0,0 +1,86 @@
+// tslint:disable:max-file-line-count
+import { InvalidAPIKeyError, NotFoundError, ValidationError, ValidationErrorCodes } from '@0x/api-utils';
+import { BigNumber } from '@0x/utils';
+import * as express from 'express';
+import * as HttpStatus from 'http-status-codes';
+
+import { CHAIN_ID, NATIVE_TOKEN_SYMBOL, NATIVE_WRAPPED_TOKEN_SYMBOL } from '../config';
+import { schemas } from '../schemas';
+import { EMPTY_QUOTE_RESPONSE, FetchIndicativeQuoteParams, RfqmService } from '../services/rfqm_service';
+import { ConfigManager } from '../utils/config_manager';
+import { schemaUtils } from '../utils/schema_utils';
+import { findTokenAddressOrThrowApiError } from '../utils/token_metadata_utils';
+
+export class RfqmHandlers {
+    constructor(private readonly _rfqmService: RfqmService, private readonly _configManager: ConfigManager) {}
+
+    public async getIndicativeQuoteAsync(req: express.Request, res: express.Response): Promise<void> {
+        const params = this._parseFetchIndicativeQuoteParams(req);
+        let indicativeQuote;
+        try {
+            indicativeQuote = await this._rfqmService.fetchIndicativeQuoteAsync(params);
+        } catch (err) {
+            req.log.error(err, 'Encountered an error while fetching an rfqm indicative quote');
+            throw new NotFoundError('Unablie to retrieve a price');
+        }
+
+        if (indicativeQuote === EMPTY_QUOTE_RESPONSE) {
```

Comment:
After our previous discussion, this can simply change to `if indicativeQuote === null` ..